pkg/filesystem/filesystem.go

```diff
@@ -431,10 +431,10 @@ func compareTars(leftFile, rightFile io.Reader) (bool, error) {
 		// Check if both archives contain the files
 		switch {
 		case len(hashes[0]) == 0:
-			logrus.Debugf("%s only exists in left tar", filename)
+			logrus.Debugf("%s does not exist in left tar", filename)
```

Comment:
Was this log statement incorrect previously? Don't see any new logic changes around it

Reply:
Before it said "the file **only** exists in **right** tar", whereas now it says "the file **no longer** exists in **left** tar"

Since we switched both the side and the meaning, it's the same log